MDANSE/Src/MDANSE/Framework/AtomSelector/atom_selection.py

```diff
@@ -0,0 +1,66 @@
+#    This file is part of MDANSE.
+#
+#    MDANSE is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+from typing import Union, Dict, Any, Set
+
+from MDANSE.Chemistry.ChemicalSystem import ChemicalSystem
+from MDANSE.MolecularDynamics.Trajectory import Trajectory
+
+
+def select_atoms(
+    trajectory: Trajectory, **function_parameters: Dict[str, Any]
+) -> Set[int]:
+    """Selects all the atoms in the trajectory.
+
+    Parameters
+    ----------
+    selection : Set[int]
+        A set of atom indices
+    trajectory : Trajectory
+        A trajectory instance to which the selection is applied
+
+    Returns
+    -------
+    Set[int]
+        Set of all the atom indices
+    """
+    selection = set()
+    system = trajectory.chemical_system
+    element_list = system.atom_list
+    name_list = system.name_list
+    indices = set(range(len(element_list)))
+    index_list = function_parameters.get("index_list", None)
+    index_range = function_parameters.get("index_range", None)
+    index_slice = function_parameters.get("index_slice", None)
```

Comment:
`None` is default, but happy to leave it for clarity. In fact, given you're subsequently doing intersections on these, may want to return empty `set`s.
```suggestion
    index_list = function_parameters.get("index_list")
    index_range = function_parameters.get("index_range")
    index_slice = function_parameters.get("index_slice")
```